### ISSUE TITLE
convoluation1D input_shape questions

### ISSUE THREAD

**Issue by MagicYoung** [fchollet/keras#2006]
The `input_shape` was predefined as `(samples, steps, input_dim)`. What is the meaning of `steps` here? And along which dimension will the linear convolution operation perform?

I have a training data with the shape `(samples, dims)`. I want to perform convolution operations along the `dims` axis. How can I define the `input_shape` for the `convolution1D` as the first layer? 

Thanks

Please make sure that the boxes below are checked before you submit your issue. Thank you!

- [x] Check that you are up-to-date with the master branch of Keras. You can update with:
pip install git+git://github.com/fchollet/keras.git --upgrade --no-deps

- [x] If running on Theano, check that you are up-to-date with the master branch of Theano. You can update with:
pip install git+git://github.com/Theano/Theano.git --upgrade --no-deps

- [ ] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short).

**Comment by lukovkin** [comment]
Convolution1D will be performed along steps. Steps correspond to the length of 1D sequence, 'steps' are 1D variant of Convolution2D nb_rows, nb_cols.
If in your case the sequence has a length of 'dims', you should define input_shape as (samples, dims, 1) - if you have only one feature.

**Comment by MagicYoung** [comment]
@lukovkin Thanks for your clear answer.
